providers.tf

```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+  alias = "glue"
+}
```

Comment:
is this definition still needed even though you configured this alias already in versions.tf?